02_cancer_type_classification/lasso_range_analysis/nbconverted/lasso_range_gene.py

```diff
@@ -317,7 +317,8 @@ def add_smallest_vline(data, **kws):
     ]
     legend_labels = ['"best"', '"smallest good"']
     l = ax.legend(legend_handles, legend_labels, title='Model choice',
-                  loc='lower left', bbox_to_anchor=(2.28, 1.3))
+                  # loc='lower left', bbox_to_anchor=(2.28, 1.3))
```

Comment:
Do you want to get rid of the commented code?

Reply:
I think I want to keep it, but I'll add an explanation for why it's there